client/task-list/index.js

```diff
@@ -173,6 +180,14 @@ const TaskDashboard = ( { userPreferences, query } ) => {
 		recordEvent( 'tasklist_click', {
 			task_name: taskName,
 		} );
+		if ( visitedTasks && ! visitedTasks.includes( taskName ) ) {
+			updateOptions( {
```

Comment:
Not sure if there's an easy way around this, but `updateOptions` represents technical debt we'll need to resolve in the future.

Probably cannot be resolved easily in this PR, but we should think about how we handle tracking this kind of information in the future.

Reply:
Hey @joshuatf, Pedro [wants](https://github.com/woocommerce/woocommerce-admin/issues/7567#issuecomment-908940392) to release this change in the 2.6.x version. Since @louwie17 is AFK today and tomorrow, I made a change in https://github.com/woocommerce/woocommerce-admin/pull/7586/commits/83e6574ef250dd8c65bab7f949147a5abce9ce5f

I'm not sure if this is the right way, so please feel free to change anything you would like.

The change in the commit updates`woocommerce_task_list_tracked_completed_actions` option when an extension is installed or activated on the marketing task page. The value is used to determine `completed` status. 

I also used `updateOptions` in the change. 

Updated testing instructions:

1. Load this branch and start a new store.
2. Finish the onboarding flow, and install a marketing plugin through (**Plugins > Add New** NOT the marketing task) You can install MailPoet or Mailchimp.
3. Go to WooCommerce > Home and notice how the Set up marketing tools shows as uncompleted.
4. Visit the marketing task and notice how your marketing plugin shows as installed (do nothing)
5. Go back to **WooCommerce > Home** and notice the marketing task is still shown as uncompleted.
6. Visit the marketing task again and install one of the recommended extensions.
7. Go back to **WooCommerce > Home** and notice the marketing task shows as completed.